compiler/test/suites/exceptions.re

```diff
@@ -27,13 +27,20 @@ describe("exceptions", ({test, testSkip}) => {
   );
   assertRunError(
     "throw_exception_2",
-    "exception HorribleError(String, Bool, Number); let _ = throw HorribleError(\"oh no\", false, 1/3)",
+    {|
+      exception HorribleError(String, Bool, Number)
+      let _ = throw HorribleError("oh no", false, 1/3)
+    |},
     "HorribleError\\(\"oh no\", false, 1/3\\)",
```

Comment:
Should we be using the alternative string syntax for these "expected" outputs too? Unless I'm mistaken, we don't need to escape `(` and `)` in the alternative string syntax (which is for regexp stuff in standard strings).

Reply:
I looks like the `\` is what's escaped, but I', not sure why. But I suppose it would be good to do these too to avoid the escapes.